opencti-platform/opencti-graphql/src/domain/container.js

```diff
@@ -72,15 +72,16 @@ export const objects = async (context, user, containerId, args) => {
   }
   // Rebuild the final result
   const resultNodes = [];
-  for (let index = 0; index < elements.length; index += 1) {
+  const elements = edges?.map((n) => n.node);
+  for (let index = 0; index < elements?.length; index += 1) {
     const element = elements[index];
     const relationTypes = relationsMap.get(element.id);
     if (relationTypes) {
       const edge = { types: relationTypes, node: element, sort: element.sort };
       resultNodes.push(edge);
     }
   }
-  return buildPagination(args.first, args.after, resultNodes, resultNodes.length);
+  return buildPagination(pageInfo?.startCursor, pageInfo?.endCursor, resultNodes, pageInfo?.globalCount);
```

Comment:
why `pageInfo?.startCursor` instead of `args.first` ? and same for after ?
The first argument of `buildPagination` is `limit`, should be the number of elements we get for each page, I don't think it's `startCursor`

Reply:
fix